installers/linux-shared/after-remove.sh.erb

```diff
@@ -17,9 +17,9 @@
 -%>
 
 <% if type == 'rpm' -%>
-    rm -rf /var/lib/<%= name %> /var/log/<%= name %> /etc/defaults/<%= name %>
+    rm -rf /var/lib/<%= name %> /var/log/<%= name %>
 <% elsif type == 'deb' -%>
     update-rc.d <%= name %> remove
-    rm -rf /var/lib/<%= name %> /var/log/<%= name %> /etc/defaults/<%= name %>
+    rm -rf /var/lib/<%= name %> /var/log/<%= name %>
```

Comment:
Whether it is rpm or deb, ```rm -rf /var/lib/<%= name %> /var/log/<%= name %>``` is the same for both. Perhaps you could refactor this?